package.json

```diff
@@ -1,7 +1,7 @@
 {
   "scripts": {
     "pre": "lerna clean --yes && lerna bootstrap && lerna run build",
-    "postinstall": "dooboolab-welcome postinstall && cp -R .storybook storybook",
+    "postinstall": "dooboolab-welcome postinstall && rm -rf storybook && cp -R .storybook storybook",
```

Comment:
@JeffGuKang So did you experience that the `cp -R` is not overwritting `storybook`?

Reply:
![image](https://user-images.githubusercontent.com/27461460/85572708-30d3ca00-b670-11ea-84d4-5403b56d69e5.png)

 The changes are not what you've written in the `PR`.

Reply:
It makes another folder in `storybook` if there is storybook folder already. 
```
storybook
 L storybook
```

Reply:
@hyochan I updated description. It was written before force push.